learn/experimental/overview.mdx

```diff
@@ -0,0 +1,40 @@
+
+An experimental feature's API can change significantly and become incompatible between releases. Meilisearch does not recommend using experimental features in a production environment.
+
+Meilisearch makes experimental features available expecting they will become stable in a future release. However, it is not possible to guarantee when and if this will happen.
+
+## Configuring experimental features
+
+There are two main ways to activate and deactivate experimental features. Some must be configured at launch with a command-line flag or an environment variable. Others can only be configured with the `/experimental-features` API route.
+
+## Configuring experimental features at launch
+
+Launch Meilisearch using [configuration options](/learn/configuration/instance_options) to activate an experimental feature:
+
+```sh
+./meilisearch --experimental-enable-metrics
+```
+
+Consult the documentation for the specific features you are interested in to find their respective options. Command-line flags for experimental features are always prefixed with `--experimental`. Environment variables for experimental features are always prefixed with `MEILI_EXPERIMENTAL`.
+
+Activating or deactivating experimental features this way requires you to relaunch Meilisearch.
+
+### Configuring experimental features during runtime
+
+[Use the `/experimental-features` API route](/reference/api/experimental) to configure experimental features during runtime:
+
+<CodeSamples id="update_experimental_features_1" />
+
+Activating or deactivating experimental features this way does not require you to relaunch Meilisearch.
+
+## Current experimental features
+
+| Name                                                                             | Description                                                | How to configure                                  |
+|----------------------------------------------------------------------------------|------------------------------------------------------------|---------------------------------------------------|
+| [Metrics API](/learn/experimental/metrics)                                       | Exposes Prometheus-compatible analytics data               | At launch with a CLI flag or environment variable |
+| [Ranking score details](/learn/experimental/ranking_rule_score_details)          | Exposes advanced result ranking score data                 | During runtime with the API route                 |
+| [Reduce indexing memory usage](/learn/experimental/reduce-indexing-memory-usage) | Optimizes indexing performance                             | At launch with a CLI flag or environment variable |
+| [Vector search](/learn/experimental/vector_search)                               | Allows Meilisearch to function as a vector embedding store | During runtime with the API route                 |
```

Comment:
Seems like you could simplify the value of the last column of the table: either "CLI flag or env variable" or "`/experimental-features` API route"

I would caution against using this phrase "during runtime." I think the word runtime is more often used in the set phrase "at runtime", which is itself ambiguous as it can mean at launch or at any point in the program's execution.

In general, I also think this concept of "runtime" is more common with programs that follow a fairly set process of execution, rather than server-based APIs that listen for calls. TL;DR I would just avoid the phrase "during runtime" if possible since it is ambiguous.